### PR TITLE
Readme and JSDoc changes

### PR DESCRIPTION
**Pull request by evelynhathaway**
Mainly small typos and grammar changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ Empty the Calender.
 
 ### Event
 
-#### uid([_String_|_Number_ uid]) or id([_String_|_Number_ id])
+#### uid([_String_ uid]) or id([_String_ id])
```
Comment (evelynhathaway) [inline README.md]: Oops, I didn't know the `uid` change slipped into this PR.

`uid` has multiple type definitions that describe it only returning and accepting strings, but now that I look at the tests and code, I'm guessing the typings are the one needing updating. I will revert this line and open an issue for discussion.

File: README.md [hunk 2]
```diff
@@ -220,19 +220,19 @@ cal.events(); // --> [ICalEvent]
 ```
 
 
-#### save(**_String_ file**[, _Function_ cb])
+#### save(**_String_ file**, _Function_ cb)
```
Comment (evelynhathaway) [inline README.md]: Optional callback for `fs.writefile()` was deprecated in Node `v7.0.0` and errors in `v10.0.0`.
See the history section in [fs.writeFile's docs](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback)

File: README.md (hunk 1 above)
Comment (evelynhathaway) [reply README.md]: Opened #133

File: README.md (hunk 2 above)
Comment (sebbo2002) [reply README.md]: Oops, that totally passed me by. Thank you very much. 🙈

**Comment by sebbo2002** [comment]
Great work, thank you! ❤️

**Pull request merged**